samples/src/Microsoft.Azure.IIoT.App/src/Pages/_DrawerDiscoverer.razor

```diff
@@ -18,11 +18,11 @@
     <div class="list-group-item text-left">
         <form>
             <div class="form-group">
-                <label>Mask address range (CIDR notation)</label>
+                <label>Mask Address Range (CIDR notation)</label>
```

Comment:
I think Maida just changed it to lowercase - please sync with her and Daniele wrt the changes.

Reply:
This fix is in agreement with Maida and Daniele